app/domain/errors.rb

```diff
@@ -349,13 +354,18 @@ module AuthnGcp
       InvalidAudience = ::Util::TrackableErrorClass.new(
         msg:  "'audience' token claim {0} is invalid. The format should be " \
               "'conjur/<account_name>/<host_id>'",
-        code: "CONJ00066E"
+        code: "CONJ00067E"
       )
 
       TokenClaimNotFoundOrEmpty = ::Util::TrackableErrorClass.new(
         msg:  "Claim '{0-claim-name}' not found or empty in token. " \
               "Verify you request the token using 'format=full'",
-        code: "CONJ00067E"
+        code: "CONJ00068E"
+      )
+
+      RoleMissingRequiredConstraints = ::Util::TrackableErrorClass.new(
+        msg:  "Role should have at least one of the following constraints: {0-constraints}",
```

Comment:
will constraints have the `authn-gcp` prefix? do we want them to be present?

Reply:
Yes they will be